### PR TITLE
UI fixes and enhancements

### LINKED ISSUE #82: UI: Reminder ListView not resizing in TargetPersonCard properly
ezgif-5-aa685f2f2d

### Intended behaviour

The Reminder ListView should resize appropriately so that the whole `TargetPersonCard` is not scrollable.

This will require a refactor of the UI code such that `TargetPersonPanel` is no longer a `ListView`. Requires some research first.

### PR DESCRIPTION
**Pull request by Polygonalr**
- [x] Create a separate UI component for Quotes instead of appending it to welcomeMessage
- [X] Fix double reminderlist in TargetPersonPanel
- [X] Fix #82 by updating TargetPerson to not use an `ObservableList` and `ListView` (as they have limited functionality when displaying only 1 object, which is the case for TargetPerson.
- [x] Update all `ListView` to show "No entries found" when the corresponding `ObservableList` has no objects.
- [x] Remove remarks from `PersonListCard`

### CODE REVIEW

**Review by malwaregarry — COMMENTED** [review]
LGTM so far

File: src/main/java/seedu/address/model/person/TargetPerson.java [hunk 1]
```diff
@@ -44,7 +82,12 @@ public Person get() {
      */
     public void clear() {
         targetPerson = Optional.empty();
-        targetPersonList.clear();
+        name.set("");
+        phone.set("");
+        address.set("");
+        email.set("");
+        remark.set("");
+        tags.clear();
```
[the remark this answers is not in the document]
Comment (Polygonalr) [reply src/main/java/seedu/address/model/person/TargetPerson.java:90]: Yea I should probably do that.

**Review by malwaregarry — APPROVED** [review]
LGTM

**Review by weiquany — APPROVED** [review]
LGTM

**Pull request merged**